resources/META-INF/plugin.xml

```diff
@@ -198,6 +198,10 @@
     <localInspection language="go" displayName="Unused parameter inspection" groupPath="Go"
                      groupName="Declaration redundancy" enabledByDefault="true" level="WARNING"
                      implementationClass="com.goide.inspections.unresolved.GoUnusedParameterInspection"/>
+
+    <localInspection language="go" displayName="Empty declaration inspection" groupPath="Go"
+                     groupName="Declaration redundancy" enabledByDefault="true" level="WARNING"
```

Comment:
I think this should be `WEAK_WARNING` since the code compiles and runs ok with empty declarations. Should I submit a PR to change ti?